scalajslib/src/mill/scalajslib/ScalaJSModule.scala

```diff
@@ -293,6 +296,9 @@ trait ScalaJSModule extends scalalib.ScalaModule { outer =>
   /** Whether to emit a source map. */
   def scalaJSSourceMap: T[Boolean] = Task { true }
 
+  /** Whether to emit WASM. As of Nov 2024 scala JS wasm support is experimental */
```

Comment:
You can borrow some of the docs from scala.js: https://github.com/scala-js/scala-js/blob/b988843fd3718c26bcc41313a56632efdad1d616/linker-interface/shared/src/main/scala/org/scalajs/linker/interface/StandardConfig.scala#L189-L218

Reply:
👍